tests/ai/dataset_pipeline/test_voice_quality_assessor.py

```
@@ -2,29 +2,32 @@
 from pathlib import Path
 from pydub import AudioSegment  # type: ignore[import]
 from ai.dataset_pipeline import voice_quality_assessor as vqa
 from pytest import TempPathFactory
 
+
 def create_test_audio(
-    duration_ms: int = 3000,
-    silence_ms: int = 0,
-    snr_db: int = 20,
-    clipping: bool = False
+    duration_ms: int = 3000, silence_ms: int = 0, snr_db: int = 20, clipping: bool = False
```

Ruff (0.11.9)

Unused function argument: `snr_db`

(ARG001)